### ISSUE TITLE
Feature Request: onSkip actions override

### ISSUE THREAD

**Issue by math-muller** [RafaelBarbosatec/tutorial_coach_mark#89]
Make it possible to put for example a next button in place of the skip button.

**Comment by UlyssesAlves** [comment]
Yes. I tried to make skip go to the next step, but it finishes the whole tutorial. I

t would be great to be able to override this default behaviour so that the user may have a visual reference as to where to press to go to the next step of the tutorial being displayed.

Since this is are not possible, I am trusting the user will press the overlay or the highlighted area to go on with using the app, but he could easily think the app is broken and there's nothing on screen for him to press or do do to leave the current animation.

A skip action which takes the user to the next step would be sometimes a better solution than a skip action which finishes the whole tutorial.

**Comment by murat-ti** [comment]
Hi everybody,

I made some little changes in code for changing onSkip behaviour:
https://github.com/RafaelBarbosatec/tutorial_coach_mark/pull/140/commits/f24587d39de8ccad53e7f80cee6997c098c1248c

Now if onSkip value is null, it will run default action (skip)
Othercase, you can put your own logic. For it was show next tutorial (next)
Good luck.

**Comment by RafaelBarbosatec** [comment]
Already supported: `onSkip` can return `false` to prevent skipping (the tutorial then advances to the next target instead of closing), and `skipWidget` lets you replace the skip button with any widget — e.g. a 'Next' button in place of 'SKIP'. Closing as resolved.
